content/docs/fundamentals/converting/resolving-errors@ko.md

```diff
@@ -0,0 +1,350 @@
+---
+$title: 유효성 검사 오류 해결
+$order: 2
+toc: true
+---
+
+[TOC]
+
+이 섹션에서는 AMP 페이지에 있는 AMP 유효성 검사 오류 및 해결 방법을 살펴봅니다.  콘솔에 따라 오류가 아래와 다른 순서로 나타날 수 있습니다.
+
+## 문자 집합 포함
+
+우선 다음 오류를 해결해 보겠습니다.
+
+<pre class="error-text">
+필수 'meta charset=utf-8' 태그가 누락되었거나 잘못되었습니다.
+</pre>
+
+AMP에서 텍스트를 올바르게 표시하려면 페이지의 문자 집합을 지정해야 합니다. 또한 메타 문자 집합 정보는 `<head> ` 태그의 첫 번째 하위 요소여야 합니다. 이 태그를 첫 번째로 사용해야 하는 이유는 메타 문자 집합 태그 앞에 추가된 콘텐츠를 다시 해석하는 일을 피하기 위해서입니다.
+
+다음 코드를 `<head>` 태그의 첫 번째 줄에 **추가**합니다.
+
+
+```html
+<meta charset="utf-8" />
+```
+
+파일을 **저장**하고 페이지를 새로고침합니다. 문자 집합 오류가 더 이상 표시되지 않는지 확인합니다.
+
+## 표준 링크 포함
+
+이제 다음 오류를 살펴보겠습니다.
+
+<pre class="error-text">
+필수 'link rel=canonical' 태그가 누락되었거나 잘못되었습니다.
+</pre>
+
+모든 AMP 문서에는 해당 문서의 '표준' 버전을 참조하는 링크가 있어야 합니다.  이 가이드에 포함되어 있는 [페이지를 검색할 수 있도록 설정](/ko/docs/fundamentals/converting/discoverable.html) 단계에서 표준 페이지란 무엇이며 표준 연결에 사용할 수 있는 접근방식으로는 어떤 것이 있는지 자세히 알아보겠습니다.
+
+이 가이드에서는 변환 중인 원본 HTML 문서를 표준 페이지로 간주합니다.
+
+계속해서 `<meta charset="utf-8" />` 태그 아래에 다음 코드를 **추가**합니다.
+
+```html
+<link rel="canonical" href="/article.html">
+```
+
+{% call callout('참고', type='note') %}
+독립된 표준 AMP 페이지도 만들 수 있습니다. 여전히 표준 링크는 필요하며 AMP 문서 자체를 가리켜야 합니다.
+
+```html
+<link rel="canonical" href="article.amp.html">
+```
+{% endcall %}
+
+이제 페이지를 **새로고침**합니다. 아직 수정해야 할 오류가 많지만 표준 링크 오류는 모두 사라졌습니다.
+
+## AMP 속성 지정
+
+페이지를 AMP 문서로 선언하려면 페이지의 루트 `<html>` 요소에 속성이 필요합니다.
+
+<pre class="error-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).